lib/grape/middleware/formatter.rb

```diff
@@ -19,6 +19,7 @@ def before
       end
 
       def after
+        return unless @app_response
```

Comment:
I think the semantics of `after` is to always execute, with or without `@app_response`.